xm-commons-ms-web/build.gradle

```diff
@@ -18,7 +18,8 @@ dependencies {
     }
     implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310"
     implementation "com.fasterxml.jackson.datatype:jackson-datatype-hibernate5"
+    implementation "com.fasterxml.jackson.datatype:jackson-datatype-hibernate6"
```

Comment:
why we need support both versions?

Reply:
@ivalex0027 probably not needed,  i will add comment and we will remove it before final commit

Reply:
already removed hibernate5